src/components/resource.js

```diff
@@ -10,8 +10,8 @@ import ResultUI from "./result-ui";
 import {LocationContext} from "../contexts/location";
 
 const Resource = () => {
-  const { responseDocument, onEntryPoint } = useContext(LocationContext);
-  const { describedBy: _, ...resourceLinks } = !onEntryPoint && responseDocument ? responseDocument.getLinks() : {};
+  const { responseDocument } = useContext(LocationContext);
+  const { describedBy: _, ...resourceLinks } = responseDocument ? responseDocument.getLinks() : {};
```

Comment:
Do you think there's a better pattern for this? We are just stripping out `describedBy` from the links, but we could also grab `self` for example. Not a critique just thinking this feels a little hacky and specific.